vagrant/scripts/generate-local-values-file.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+cat > /sumologic/vagrant/values.local.yaml <<END
+fluentd:
+  metadata:
+    pluginLogLevel: "info"
```

Comment:
I doubt we should add this to the auto generated file

Reply:
I think it's nice to have developer settings for the development values yaml file.

Reply:
I do not agree. every feature development requires another set of the settings and they may conflict with each other

Reply:
Removed the `pluginLogLevel` config entry.